Signal/src/view controllers/ContactsPicker.swift

```diff
@@ -66,8 +68,8 @@ open class ContactsPicker: UITableViewController, UISearchResultsUpdating, UISea
         super.viewDidLoad()
         title = NSLocalizedString("INVITE_FRIENDS_PICKER_TITLE", comment: "Navbar title")
 
-        // Don't obscure table header (search bar) with table index
-        tableView.sectionIndexBackgroundColor = UIColor.clear
+        searchBar.placeholder = NSLocalizedString("INVITE_FRIENDS_PICKER_SEARCHBAR_PLACEHOLDER", comment: "Search")
+        self.edgesForExtendedLayout = []
```

Comment:
It's not obvious what this `edgesForExtendedLayout` is doing. Can you comment the code?